assets/js/hooks/cell_editor/live_editor.js

```diff
@@ -183,11 +202,13 @@ export default class LiveEditor {
   /**
    * Updates editor selection such that cursor points to the given line.
    */
-  moveCursorToLine(lineNumber) {
+  moveCursorToLine(lineNumber, column = "1") {
     const line = this.view.state.doc.line(lineNumber);
+    const columnNumber = parseInt(column);
+    const position = line.from + columnNumber;
```

Comment:
With this change the go-to-definition jumps between the first and second character.

Instead of columns, we want to work with line + offset, where offset is just position within the line :D

Column is useful for intellisense like hover, where we want to point to a specific character.